.env.template

```diff
@@ -0,0 +1,10 @@
+SECRET_KEY = 'django-insecure-g79&1(su4lozn%*x$=dzw5b_-e#noil2i%o0bau&0+hj#2(r5l'
+DEBUG = True
+ALLOWED_HOSTS = ['*']
+TIME_ZONE = 'UTC'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'kaushik.tagline@gmail.com'
+EMAIL_HOST_PASSWORD = 'tagline@123'
+EMAIL_USE_TLS = True
+DEFAULT_FROM_EMAIL = 'ShopFashion Team <noreply@example.com>'
```

Comment:
There should be no space between the env variable and value. No single quotes 
required.